CHANGELOG.md

```
@@ -1100,6 +1139,10 @@ The minimum supported Rust version (MSRV) is now 1.51.0 (Mar 25, 2021).
 [gh-issue-0034]: https://github.com/moka-rs/moka/issues/34/
 [gh-issue-0031]: https://github.com/moka-rs/moka/issues/31/
 
+[gh-pull-0584]: https://github.com/moka-rs/moka/pull/584/
+[gh-pull-0583]: https://github.com/moka-rs/moka/pull/583/
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "gh-pull-0583"

(MD053, link-image-reference-definitions)